### ISSUE TITLE
No Keybase files in Finder since macOS Monterey upgrade

### ISSUE THREAD

**Issue by greg-raven** [keybase/client#24677]
No Keybase files in Finder since macOS Monterey upgrade. Lot's of apps broke, but Keybase is the only one I have not been able to get running properly again.

my log id: 1808db70156fcf20db6b961c

**Comment by mikwat** [comment]
Same problem.

**Comment by minoro-ben** [comment]
Same here

**Comment by RayOei** [comment]
Seems duplicate/similar to #24668 and #24636 and @24578

**Comment by marcofranssen** [comment]
Same here 👍 is this something going to be resolved?

**Comment by YourMomsAndMe** [comment]
same issue here. 

I had just opened a ticket [ will close since its a dupe now, but heres what I noted]:

```
macOS Monterey 12.0.1, Keybase v 5.8.0, macFuse 4.2.2 [just released; but earlier today I was on 4.2.1]

When you head over to KB > Files > Enable Keybase in Finder, and approve, it just spins but does not take.

Earlier today I removed KB + macFuse, and tried to set up from scratch but still not taking. I did approve in  > syspref >sec&pri > Privacy > Full Disk Access, but same. Files and Folders is already enabled once you approve Full Disk Access. But I did try just Files and Folders and nope: same, just loops and never enables.

This was working fine in Big Sur.

Anything you folks think I can try?
```

**Comment by YourMomsAndMe** [comment]
dupe, found this: https://github.com/keybase/client/issues/24677

**Comment by terminalfool** [comment]
Ditto.

**Comment by maxtaco** [comment]
Closed as duplicative of #24636
